server/organisation.go

```diff
@@ -52,19 +41,6 @@ func (a *ApplicationHandler) GetOrganisation(w http.ResponseWriter, r *http.Requ
 // @Router /api/v1/organisations [get]
 func _() {}
```

Comment:
@Youngestdev, You can take this one out too.

Reply:
@danvixent isn't this the one you added, I can't remember why again

Reply:
@Subomi @Youngestdev please we shouldn’t remove those.